### PR TITLE
Move out setting of validation rules enricher from theme container

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Field/Base/DateTimeInputField.php [hunk 1]
```diff
@@ -143,7 +142,10 @@ final public function disabled(bool $disabled = true): static
     protected function beforeRender(): void
     {
         if ($this->enrichFromValidationRules) {
-            $this->enrichment = ThemeContainer::getValidationRulesEnrichment($this, $this->getInputData());
+            $this->enrichment = $this
+                ->validationRulesEnricher
+                ?->process($this, $this->getInputData()->getValidationRules())
+                ?? [];
```
Comment (arogachev) [inline src/Field/Base/DateTimeInputField.php:148]: Need to improve coverage.

File: docs/guide/en/validation-rules-enrichment.md [hunk 2]
```diff
@@ -52,7 +52,10 @@ final class MyTextField extends InputField
     protected function beforeRender(): void
     {
         if ($this->enrichFromValidationRules) {
-            $this->enrichment = ThemeContainer::getValidationRulesEnrichment($this, $this->getInputData());
+            $this->enrichment = $this
+                ->validationRulesEnricher
+                ?->process($this, $this->getInputData()->getValidationRules())
```
Comment (arogachev) [inline docs/guide/en/validation-rules-enrichment.md:57]: Need to fix mutants.

Comment (arogachev) [reply docs/guide/en/validation-rules-enrichment.md:57]: #352

**Review by vjik — CHANGES_REQUESTED** [review]
Need way to globally set enricher.

**Comment by arogachev** [comment]
@vjik Added support to set enricher globally via theme.

**Review by vjik — APPROVED** [review]
Remove `validationRulesEnricher` from `ThemeContainer` docs.

**Review by vjik — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Pull request merged**